_data/simple-icons.json

```diff
@@ -10342,7 +10343,7 @@
         {
             "title": "Qwiklabs",
             "hex": "F5CD0E",
-            "source": "https://www.qwiklabs.com"
+            "source": "https://www.cloudskillsboost.google"
         },
```

Comment:
I think we can safely remove/rename this icon as of v10, as Qwiklabs now entirely redirects to Google Cloud Skills Boost.
I would leave quiklabs.com present as part of this PR, and open a separate one for the removal of the brand in v10.

Reply:
The icon of quicklabs is still present as an SVG on the footer of the new page.

Reply:
So it is, excuse my blindness. Ignore this comment then!